examples/working_with_fields.jl

```diff
@@ -0,0 +1,18 @@
+using Pkg; Pkg.activate("..")
+
+using Oceananigans, PyPlot
+
+import PyPlot: Plot
+
+zplot(c::CellField, args...; kwargs...) = plot(znodes(c), data(c), args...; kwargs...)
```

Comment:
I forgot that there may not be a `data` function for getting `c.data`. This line may have to be changed.

Reply:
Although, there should be a `data` function that returns a view over interior points --- this is important given that ghost cells are coming (are here)?

Reply:
Ghost cells (halo regions) are still in PR #167 until I have time to polish them off.

But yes then it makes sense to maybe define a `data(::Field)` function that returns the actual data (i.e. the interior points). I'll make a note on the PR.

Reply:
Alright, so this PR will wait for #167.